_posts/en/2020-02-16-what-devops-looks-like.md

```diff
@@ -0,0 +1,116 @@
+
+[2006 AG report](https://www.oag-bvg.gc.ca/internet/English/parl_oag_200611_03_e_14971.html): 7 large IT projects were evaluated across 4 criteria
+
+- **Governance**: Do the processes used by the government to approve and manage large IT projects increase the project's likelihood of success?
+- **Business case**: Did the department or agency proposing the project clearly define the business needs it expected the project to meet?
+- **Organizational capacity**: Did the department have people with the needed skills and experience to manage a large IT project and did the organization have the ability to use all of a system's capabilities to improve the way it does business?
+- **Project management**: Did the department follow accepted best practices in managing the project?
+
+Here are some of the findings:
+
+- The quality of governance varied widely from project to project. In four of the seven projects we found that governance responsibilities were not carried out adequately because key issues that impacted project performance were either not reported or not resolved.
+- Five of the seven projects we looked at were allowed to proceed with a business case that was incomplete or out-of-date or contained information that could not be supported.
+- Four of the projects were undertaken by departments that lacked the appropriate skills and experience to manage the projects or the capacity to use the system to improve the way they deliver their programs.
+- Depending on the project, the quality of project management ranged from good to seriously flawed. In two cases, poor project management led to long delays and large cost overruns.
+
+[2011 AG report](https://www.oag-bvg.gc.ca/internet/English/parl_oag_201106_02_e_35370.html): Examination of the progress made from the 2006 recommendations
+
+Here are some of the findings:
+
+- Overall, the government has made unsatisfactory progress on its commitments in response to our 2006 recommendations.Although some improvements have been made, progress has been unsatisfactory in the important areas of governance and project business cases. Only two of the five projects we looked at, the Temporary Resident Biometrics Project and Integrated Revenue Collections systems, met most of our criteria for well-managed projects.
+- In order to increase the likelihood of success, departments have significantly reduced the scope of the projects and considerably extended their timelines.In the area of project governance, the Expenditure Management Information System and Global Case Management System have had important deliverables deferred without full analysis of the impacts and costs of not completing these projects. In three of the five projects examined, the project business cases did not identify measurable benefits or the benefits have not been measured.For example, the 2007 business case for the Secure Channel no longer quantifies the financial benefits.
```

Comment:
Is there a space missing between the period and 'In'?  Or is there one space too many between the period and the second 'In'?  Is there a space missing between the period and 'For example'?